landlab/components/erosion_deposition/erosion_deposition.py

```diff
@@ -8,14 +8,42 @@
 TIME_STEP_FACTOR = 0.5  # factor used in simple subdivision solver
 DEFAULT_MINIMUM_TIME_STEP = 0.001  # default minimum time step duration
 
+
 class ErosionDeposition(_GeneralizedErosionDeposition):
     """
-    Erosion-Deposition model in the style of Davy and Lague (2009)
+    Erosion-Deposition model in the style of Davy and Lague (2009). It uses a
+    mass balance approach across the total sediment mass both in the bed and
+    in transport coupled with explicit representation of the sediment
+    transport lengthscale (the "xi-q" model) to derive a range of erosional
+    and depositional responses in river channels.
+
+    This implementation is close to the Davy & Lague scheme, with a few
+    deviations:
+    - Sediment porosity is handled explicitly in this implementation.
+    - A fraction of the eroded sediment is permitted to enter the wash load,
+      and lost to the mass balance (F_f).
+    - Here an incision threshold is permitted, where it was not by Davy &
+      Lague. It is implemented with an exponentially smoothed form to prevent
+      discontinuities in the parameter space. See the
+      StreamPowerSmoothThresholdEroder for more documentation.
+    - This component uses an "effective" settling velocity, v_s, as one of its
+      inputs. This parameter is simply equal to Davy & Lague's `d_star * V`
+      dimensionless number.
+
+    Erosion of the bed follows a stream power formulation, i.e.,
+
+    E = K * q ** m_sp * S ** n_sp - optional threshold (see above)
+
+    Note that in this implementation v_s becomes the parameter that
+    fundamentally controls response style. Very small v_s will lead to a
+    detachment-limited response style, very large v_s will lead to a
+    transport-limited response style. v_s == 1 means equal contributions from
```

Comment:
I'm not sure about this interpretation of v_s = 1. You still have drainage area (or discharge), collectively forming a length scale.

Reply:
Bump @cmshobe ... Good to get this in ASAP before it just atrophies.

Reply:
@SiccarPoint I am so sorry!! On it now

Reply:
@SiccarPoint @gregtucker:

The deal here is that the controlling dimensionless quantity is v_s/r, where r is the runoff rate from Q=rA. When v_s/r = 1, contributions from the DL and TL regimes are equal. When that ratio gets huge, we go to the TL regime, and vice versa. The reason I told Dan that v_s was the control is that in the Landlab implementation as far as I know we don't allow r to be a free parameter.

I'm going to fix the docs right now to make this clear.